test/parallel/test-console.js

```diff
@@ -187,3 +188,15 @@ common.hijackStderr(common.mustCall(function(data) {
     assert.strictEqual(data.includes('no such label'), true);
   });
 }));
+
+const pristineConsole = vm.runInNewContext('this.console');
+for (const name in console.inspector) {
+
+  // No functions on console that do nothing when no inspector is attached
+  if (name in console) {
+    assert.notStrictEqual(console[name], console.inspector[name]);
+  }
+
+  // Everything in a v8 console object should be in console.inspector
```

Comment:
Aren't you testing the other direction (i.e. that everything that's on `console.inspector` also exsists on the `pristineConsole` (i.e. the V8 native)?
So AFAICT the comment should be:
```js
// Everything in `console.inspector` should also be in a v8 pristine console.
```